### PR TITLE
STY: Increase readability slightly

### PR DESCRIPTION
**Pull request by j-t-1**

### CODE REVIEW

File: pypdf/_writer.py [hunk 1]
```diff
@@ -497,13 +497,13 @@ def _add_page(
         else:
             cast(ArrayObject, node[PA.KIDS]).append(page.indirect_reference)
             self.flattened_pages.append(page)
-        cpt = 1000
+        cnt = 1000
```
Comment (stefan6419846) [inline pypdf/_writer.py:500]: Are we able to find an even better name here? For me, neither `cpt` nor `cnt` are really self-explaining.

Comment (j-t-1) [reply pypdf/_writer.py]: Have changed to "recurse", there may be better.

File: pypdf/_writer.py [hunk 2]
```diff
@@ -1058,7 +1058,7 @@ def update_page_form_field_values(
             raise PyPdfError("No /AcroForm dictionary in PdfWriter Object")
         af = cast(DictionaryObject, self._root_object[CatalogDictionary.ACRO_FORM])
         if InteractiveFormDictEntries.Fields not in af:
-            raise PyPdfError("No /Fields dictionary in Pdf in PdfWriter Object")
+            raise PyPdfError("No /Fields dictionary in PdfWriter Object")
```
Comment (stefan6419846) [inline pypdf/_writer.py:1061]: Could you please elaborate the reason for this change?

Comment (j-t-1) [reply pypdf/_writer.py:1061]: More similar to line 1058, and the PDF and the PdfWriter Object are equivalent.

Comment (stefan6419846) [reply pypdf/_writer.py:1061]: Yes and no. Strictly speaking, a PdfWriter holds a custom representation of the actual PDF structure. Nevertheless, this change is unrelated to the readability aspects IMHO.

**Review by stefan6419846 — APPROVED** [review]

**Pull request merged**